README.md

```diff
@@ -1,269 +1,7 @@
-
-#### `config/`
-
-The `config` directory contains the config files needed for the electron app.  This includes the event-marker details and event codes.
-
-Note: the productId can be overwritten by the environment variable EVENT_MARKER_PRODUCT_ID
-
-### `src/`
-
-This folder contains the code for the app, the vast majority of changes and code should go here.
-
-#### `App.js`
-
-This is the starting point for the app. The `<Experiment>` component initializes a `jspsych` experiment. This is also where communication is set up with the `electron` and `psiturk` processes.
-
-#### `App.css`
-
-This is where styling for the app is housed. If colors, fonts, spacing, etc. need to be set, do it here.
-
-#### `assets/`
-
-This folder contains any static files that are used by the app, such as images.
-
-#### `config/`
-
-In the `config/` directory, there are `.js` files which contain settings for the different parts of the task.  Every task should have a `main` config and a `trigger` config (assuming use of the event marker). The `main` config has all global settings for the task (such as whether it is in mturk mode or not), load the appropriate language file, and set up a default (or only) configuration object for the task. The `trigger` config has settings specific to the event marker and uses a slightly different style of javascript as it is imported both in the React app as well as the electron process.
-
-Other config files can be used to add settings for specific blocks or sub-sections of the experiment.
-
-#### `language/`
-
-Any language that is displayed in the experiment should be stored in this folder. Usage of language json files allows for easy internationalization of the task (e.g. english and spanish) as well as allows for mturk specific language. This also makes it easy to re-use common phrases in many places in the task.
-
-#### `lib/`
-
-The `lib/` directory contains utility functions and markup that is used in the tasks.  This allows for functions and html to be re-used wherever needed. The `lib/utils.js` file contains functions that are generally useful across many tasks, whereas `lib/taskUtils.js` contains functions specific to this task.
-
-#### `lib/markup`
-
-`markup` files should contain primarily templates for HTML that is used throughout the task. Typically this will be a function that takes in some parameters and then returns a string with html.
-
-#### `timelines`
-
-`jspsych` uses `timelines` to control what `trials` are displayed in what order.  `timelines` can contain other `timelines`, which is why there may be several files in this directory.  The `main.js` file should have the timeline that is called by `App.js`.
+[![DOI](https://zenodo.org/badge/201963539.svg)](https://zenodo.org/badge/latestdoi/201963539) [![docs](https://img.shields.io/badge/docs-stable-blue)](https://brown-ccv.github.io/honeycomb-docs/)
```

Comment:
Should we add badges to the builds or the GitHub.io sample task? Although that one is not yet up because I think we need to tag. So maybe this is fine for now but we could see if after tagging we should add more badges